src/app.js

```diff
@@ -1 +1,38 @@
 'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+function copyFile() {
+  const terminalParameters = process.argv.slice(2);
+  const copyFromFile = terminalParameters[0];
+  const copyToFile = terminalParameters[1];
+
+  if (!copyFromFile || !copyToFile) {
+    global.console.log(
```

Comment:
Explicitly adding `global` to `global.console.log` is not necessary.
```suggestion
    console.log(
```

Reply:
I do it just for linter. I don't like red wave under word)